platform/security/security-services-app/src/main/resources/features.xml

```diff
@@ -331,4 +325,9 @@
         <feature>security-sts-realm</feature>
     </feature>
 
+    <feature name="security-services-app" install="auto" version="${project.version}"
+             description="The Security Application provides Authentication, Authorization, and Auditing services.\nThey comprise both a framework that developers and integrators can extend and also a reference implementation that can be used which meets security requirements.::Security">
```

Comment:
✏️can we get rid of the `::Security` now? same comment for all the apps really.

Reply:
Although the backend no longer uses the end of the description, the frontend is parsing it to give it a display name. I originally removed them but the app names were no longer user friendly. I was going to add a displayName field and method to the Application interface until I started deprecating it.

Reply:
❗️ This still seems like the last hack that couples features to apps.  Ideally this should be removed in this PR if possible but a follow up ticket would also be acceptable if it will be too much.

Reply:
I thought this comment was left on the app.json files. I should be able to update the description of this feature in this pr without any additional changes :+1: